src/org/zaproxy/zap/extension/pscanrules/ZapAddOn.xml

```diff
@@ -6,7 +6,8 @@
 	<url/>
 	<changes>
 	<![CDATA[
-	Minor code changes.
```

Comment:
The version number should be updated and old _changes_ removed (version 13 was released with ZAP 2.4.1).